### PR TITLE
Chore: Remove SWA integration.

### PR DESCRIPTION
**Pull request by jjtang1985**
This PR removes the function of pushing data to SWA.
After that, you will not see the small window for consent any more.

### CODE REVIEW

File: static/api/v1/classes/sap_cloud_sdk_core.AnyField.html [hunk 1]
```diff
@@ -27,4 +26,4 @@
...
...type-alias tsd-has-type-parameter"><span class="tsd-kind-icon">Type alias with type parameter</span></li></ul><ul class="tsd-legend"><li class="tsd-kind-interface"><span class="tsd-kind-icon">Interface</span></li><li class="tsd-kind-interface tsd-has-type-parameter"><span class="tsd-kind-icon">Interface with type parameter</span></li></ul><ul class="tsd-legend"><li class="tsd-kind-class"><span class="tsd-kind-icon">Class</span></li><li class="tsd-kind-class tsd-has-type-parameter"><span class="tsd-kind-icon">Class with type parameter</span></li></ul><ul class="tsd-legend"><li class="tsd-kind-property tsd-parent-kind-class tsd-is-inherited"><span class="tsd-kind-icon">Inherited property</span></li><li class="tsd-kind-method tsd-parent-kind-class tsd-is-inherited"><span class="tsd-kind-icon">Inherited method</span></li></ul><ul class="tsd-legend"><li class="tsd-kind-enum"><span class="tsd-kind-icon">Enumeration</span></li></ul></div><h2>Settings</h2><p>Theme <select id="theme"><option value="os">OS</option><option value="light">Light</option><option value="dark">Dark</option></select></p></div></footer><div class="overlay"></div><script src="../assets/main.js"></script></body></html>
```
Comment (jjtang1985) [inline static/api/v1/classes/sap_cloud_sdk_core.AnyField.html:29]: This seems to be related to the empty line

**Review by marikaner — APPROVED** [review]
yay

**Review by cschubertcs — COMMENTED** [review]
Looks good in general, just a quick question: Did you need to do those changes manually or is that generated code?

**Comment by jjtang1985** [comment]
>Did you need to do those changes manually or is that generated code?

I guess that should be added by the script:
https://docusaurus.io/docs/api/docusaurus-config#scripts

However, I'm not sure whether the removal is also covered, as it did not work for me.
So I did a search/replace to the removal via the IDE.

**Pull request merged**